backend/app/pydantic_models/question.py

```
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)